modules/tinymce/src/core/test/ts/browser/content/EditorContentTest.ts

```diff
@@ -71,7 +72,12 @@ describe('browser.tinymce.core.content.EditorContentTest', () => {
     const editor = hook.editor();
     editor.setContent('<p>Text to be retrieved</p>');
     const text = editor.getContent({ format: 'text' });
-    Assertions.assertHtml('Should be expected text', 'Text to be retrieved', text);
+    // TODO: <Jira> The table plugin code has been moved to core so there is now always an extra DIV in the dom for the table resize bars
```

Comment:
Oh this is a fun bug we didn't realise was happening... anyways we need a Jira number here still.